hasura/config.yaml

```diff
@@ -0,0 +1,20 @@
+version: 3
```

Comment:
Did you mean to push these hasura files to the same branch? Just wondering!

Reply:
Oops, I added them to keep my branch updated but realized now it will conflict with Signe's hasura database PR. I will remove the directory from my branch!